### DISCUSSION TITLE
"betErrorCode": "RESTRICTED"

### DISCUSSION THREAD

**Discussion by FranPetkovic** [cloudbet/docs#95] [Q&A]
My account is getting RESTRICTED error and customer support isn't much of a help

**Comment by kgravenreuth** [comment]
Hi @FranPetkovic ,

Please refer to [this discussion](https://github.com/cloudbet/docs/discussions/84) to learn more about account restricted errors. 

After reviewing your account, please note that Access from restricted countries is blocked as per [Cloudbet Terms & Conditions](https://www.cloudbet.com/en/help/terms?section=71-your-account&utm_source=github.com&utm_medium=community&utm_campaign=axis).

Note that such country restrictions are imposed both at account registration as well as bet placement times and depends on the location of your API client. This is also mentioned in our [Trading API Docs](https://www.cloudbet.com/api/?urls.primaryName=Trading&utm_source=github.com&utm_medium=community&utm_campaign=axis)

Let me know if this clarifies your question, cheers.

Regards,
Klaus Gravenreuth
Product & Engineering @ Cloudbet

**Reply by FranPetkovic** [reply]
Thanks for the info. Just wondering, is there any way to remove the restrictions on my account? I’d like to keep using Cloudbet.

**Reply by kgravenreuth** [reply]
Hi @FranPetkovic ,

As mentioned on our Terms page, these jurisdictions are restricted for betting. You are allowed to bet with your API client from an unrestricted location, given you are also physically not located in any of our restricted jurisdictions. 

Regards,
Klaus Gravenreuth
Product & Engineering @ Cloudbet
